README.md

```diff
@@ -31,22 +31,67 @@ Once installed, the following ``py.test`` command-line parameters are available:
 
 ```bash
 py.test \
-    [--ansible-inventory <path_to_inventory>] \
-    [--ansible-host-pattern <host-pattern>] \
-    [--ansible-connection <plugin>] \
-    [--ansible-module-path <path_to_modules] \
-    [--ansible-user <username>] \
-    [--ansible-sudo] \
-    [--ansible-sudo-user <username>]
+    [--inventory <path_to_inventory>] \
+    [--host-pattern <host-pattern>] \
+    [--connection <plugin>] \
+    [--module-path <path_to_modules] \
+    [--user <username>] \
+    [--become] \
+    [--become-user <username>] \
+    [--become-method <method>] \
+    [--limit <limit>] \
+    [--check]
 ```
 
 The following fixtures are available:
 
 
+### Fixture ``ansible_adhoc``
+
+The `ansible_adhoc` fixture returns a function used to initialize a `HostManager` object.
+
+```python
+def test_all_the_pings(ansible_adhoc):
+    result = ansible_adhoc().all.ping()
+    assert not result.is_failed
```

Comment:
Thanks for the catch, one needs to iterate through the result.  Will fix, thanks.